examples/notify-turn-hook.py

```
@@ -0,0 +1,220 @@
+        import urllib.request
+        data = json.dumps({
+            "message": message,
+            "speak": True,
+            "sound": True
+        }).encode()
+        req = urllib.request.Request(
+            f"{NOTIFY_SERVER}/notify",
+            data=data,
+            headers={"Content-Type": "application/json"},
+            method="POST"
+        )
```

Ruff (0.14.6)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)